spec/factories/submissions.rb

```diff
@@ -48,6 +48,7 @@
     trait :submittable do
       with_dissertation_file
       with_supplemental_files
+      with_permission_files
```

Comment:
Should this trait also include `permissions_provided { 'true' }` down below or do you think that's a better fit for the follow-up work to add the toggle?